irr/src/os.h

```diff
@@ -10,6 +10,21 @@
 #include "ILogger.h"
 #include "ITimer.h"
 
+// CODE_UNREACHABLE(): Invokes undefined behavior for unreachable code optimization
+#if defined(__cpp_lib_unreachable)
+#define CODE_UNREACHABLE() std::unreachable();
+#elif defined(__has_builtin)
+#if __has_builtin(__builtin_unreachable)
+#define CODE_UNREACHABLE() __builtin_unreachable();
+#endif
+#elif defined(_MSC_VER)
+#define CODE_UNREACHABLE() __assume(false);
+#endif
+
+#ifndef CODE_UNREACHABLE
+void CODE_UNREACHABLE [[noreturn]] ();
+#endif
```

Comment:
this function is not defined anywhere so how would this possibly work?

Reply:
Fixed in 43b8261a1